src/OrchardCore/OrchardCore.ContentManagement/Records/Migrations.cs

```diff
@@ -18,8 +18,8 @@ public Migrations(IContentDefinitionManager contentDefinitionManager)
         public async Task<int> CreateAsync()
         {
             await SchemaBuilder.CreateMapIndexTableAsync<ContentItemIndex>(table => table
-                .Column<string>("ContentItemId", c => c.WithLength(26))
-                .Column<string>("ContentItemVersionId", c => c.WithLength(26))
+                .Column<string>("ContentItemId", column => column.NotNull().WithLength(26))
+                .Column<string>("ContentItemVersionId", column => column.WithLength(26)) // Cannot be not null
```

Comment:
This is an issue as it should not allow for a NULL value. Unless we decided that a first version would have a NULL value which would be wrong.